compiler/src/main/kotlin/settings.kt

```diff
@@ -72,6 +72,11 @@ class Arguments(parser: ArgParser) {
         }
     }
 
+    val interpret by parser.flagging(
```

Comment:
What happens if I specify both `--interpret` and `--target kotlin`

Reply:
`--interpret` wins. The compiler will not compile anything.

Reply:
Wouldn't it be better if it yielded a error since an ambiguous action has been specified

Reply:
I guess. Make an issue for it.

Reply:
Issue #166 has been made